content/4_Gold/TopoSort.mdx

```diff
@@ -121,30 +121,30 @@ import java.util.*;
 import java.io.*;
 
 public class CourseSchedule {
-	public static ArrayList <Integer> g[];
-	public static ArrayList <Integer> topo = new ArrayList < Integer > ();
+	public static List<Integer>[] g;
```

Comment:
if you're gonna refactors at least rename this to something more explicit like `neighbors`

Reply:
uh, not resolved?